File: inspect_excel.js

const XLSX = require('xlsx');

const filePath = 'Performans-Data (1).xlsx';
try {
    const workbook = XLSX.readFile(filePath);
    const sheetName = workbook.SheetNames[0];
    const worksheet = workbook.Sheets[sheetName];
    const data = XLSX.utils.sheet_to_json(worksheet, { header: 1 }); // Array of arrays

    const headers = data[0];
    console.log(JSON.stringify(headers));
} catch (e) {
    console.error("Error reading file:", e);
}

File: inspect_excel.py

import pandas as pd
import sys

file_path = "Performans-Data (1).xlsx"
try:
    df = pd.read_excel(file_path, header=None)
    # Print first few rows to identify header
    print("First 5 rows:")
    print(df.head(5).to_string())
except Exception as e:
    print(f"Error reading file: {e}")
